index.js

```diff
@@ -0,0 +1,111 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const models = require('./models')
+const app = express()
+
+app.get('/movies', async (request, response) => {
+  const movies = await models.movies.findAll({
+    include: [{
+      all: true
+    }]
+  })
+  response.send(movies)
+})
+
+app.get('/movies/:identifier', async (request, response) => {
+  const movies = await models.movies.findAll({
+    include: [{
+      all: true
+    }],
+    where: {
+      id: request.params.identifier
+    }
+  })
+  response.send(movies)
+})
+
+app.get('/directors/:identifier', async (request, response) => {
+  const director = await models.directors.findAll({
+    include: {
+      model: models.movies
+    },
+    where: {
+      id: request.params.identifier
+    }
+  })
+  response.send(director)
+})
+
+app.get('/genres/:identifier', async (request, response) => {
+  const genre = await models.genres.findAll({
+    include: {
+      model: models.movies
+    },
+    where: {
+      genre: request.params.identifier
+    }
+  })
+  response.send(genre)
+})
+
+app.post('/movies', bodyParser.json(), async (request, response) => {
+  const {
+    title,
+    directors,
+    releaseDate,
+    rating,
+    runTime,
+    genres
+  } = request.body
+
+  const names = directors.split(', ')
+  const genresArray = genres.split(', ')
```

Comment:
This works but to better handle potentially dirty data you could have split on just `','` and then used trim to get rid of whitespace on the ends